### PR TITLE
build : login-page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: pages/login-page.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+            <h3 class="form-mesg">Email address</h3>
+
+            <input class="form-input" id="primary-input" type="email" name="email-id" placeholder="Enter Your Email-Id">
+
+            <h3 class="form-mesg">Password</h3>
+
+            <input class="form-input" type="password" name="password" placeholder="Enter Your Password">
+
+            <div class="checkbox-container">
```
Comment (Rockyraven) [inline pages/login-page.html:72]: remove all unused space

**Review by DivyanshSareen — APPROVED** [review]
Good to merge

File: pages/login-page.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   
+    <!----------------------------login page starts--------------------------->
+
+    
+    <!-- textarea-container is class used in main-->
+    <main class="form-container">
+        <form class="form">
+            <h2 class="form-heading">Already have an account?</h1>
+
+            <h2 class="form-sub-heading">Login</h2>
```
Comment (arjun-computer-geek) [inline pages/login-page.html:62]: Don't use h2 and h3 for no reason. It has some meaning.

File: pages/login-page.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+            <h3 class="form-mesg">Email address</h3>
+
+            <input class="form-input" id="primary-input" type="email" name="email-id" placeholder="Enter Your Email-Id">
+
+            <h3 class="form-mesg">Password</h3>
```
Comment (arjun-computer-geek) [inline pages/login-page.html:68]: Use label for that

File: pages/login-page.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+            <input class="form-input" type="password" name="password" placeholder="Enter Your Password">
+
+            <div class="checkbox-container">
+                <p>
+                <input type="checkbox" >
+                <label> Remember me </label>
```
Comment (arjun-computer-geek) [inline pages/login-page.html:75]: Use for attributes in label

**Review by Rockyraven — APPROVED** [review]
all good to me can merge

File: styles/login-page.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+.form {
+    padding: 2rem 1rem;
+    height: 80%;
+    width: 30%;
+    border: 1px solid;
+    background-color: var( --ecomm-color-form);
+    align-items: center;
+    border: 1px solid;
+    margin: auto;
+    margin-top: 6rem;
```
Comment (arjun-computer-geek) [inline styles/login-page.css:21]: For margin and padding always use em

**Review by arjun-computer-geek — APPROVED** [review]
Do all the changes. Rest all  looks good to me.😁

**Pull request merged**